dev/PackageManager/API/PackageManager.idl

```diff
@@ -0,0 +1,383 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+//import "windows.foundation.idl";
+//import "Windows.Management.Deployment.idl";
+//import "Microsoft.Windows.ApplicationModel.winmd;"
+
+//import "Windows.ApplicationModel.idl";
+import "M.AM.DynamicDependency.idl";
+//import "Microsoft.Windows.ApplicationModel.DynamicDependency.winmd";
+
+//import "Windows.Management.Deployment.idl";
+
+namespace Microsoft.Windows.Management.Deployment
```

Comment:
Presumably we'll have an API review of these coming soon?

Reply:
Yep. This is early experimental work to unblock other early folks early work. There'll be an API Spec coming soon, in a later PR based on this work plus feedback from that early use.